src/msg/async/dpdk/dpdk_rte.h

```diff
@@ -67,6 +68,7 @@ class eal {
    */
   static size_t mem_size(int num_cpus);
   static bool initialized;
+  static bool exit_thread;
```

Comment:
s/exit_thread/stopped/ 

can we take this opportunity to refactor this class to a real class in the sense that all its members are not static anymore. and make it a member of `DPDKStack`?

Reply:
ok